jolly-jellyfish/src/page_maker/urls.py

```diff
@@ -0,0 +1,42 @@
+from django.urls import path
+
+from .views import (
+    MainView,
+    UserRegister,
+    UserDetailView,
+    UserUpdateView,
+    UserDeleteView,
+    WebpageCreateView,
+    WebpageView,
+    WebpageDetailView,
+    WebpageUpdateView,
+    WebpageDeleteView,
+    WebpageListView,
+    TemplateCreateView,
+    TemplateView,
+    TemplateDeleteView,
+    CommentCreateView,
+    CommentDeleteView
+)
+
+urlpatterns = [
+    path('', MainView.as_view(), name='home'),
+    path('users/register', UserRegister.as_view(), name='register'),
```

Comment:
```suggestion
    path('/', MainView.as_view(), name='home'),
    path('users/register/', UserRegister.as_view(), name='register'),
```
Yes, technically, the other two should end with / too.